independent-projects/bootstrap/pom.xml

```diff
@@ -85,8 +86,18 @@
                         <groupId>org.sonatype.plexus</groupId>
                         <artifactId>plexus-cipher</artifactId>
                     </exclusion>
+                    <exclusion>
```

Comment:
needed because jsr250 brought in old security annotations (e.g. `@DenyAll` that couldn't be placed on classes)